nilearn/plotting/find_cuts.py

```diff
@@ -196,6 +196,10 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
     orig_data = np.abs(img.get_data())
```

Comment:
Maybe use this opportunity to set its default to the same value than in displays (looks like it's 7) from https://github.com/nilearn/nilearn/blob/de38fe03a535b79b4250794833fe4049730e7ce7/nilearn/plotting/displays.py#L939.